app/views/layouts/_static_slot_fields.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="card m-2 p-2 nested-fields">
   <div class="col-lg-12">
     <div class="field">
-      <%= f.label :test, 'static_slot' %><br />
+      <%= f.label :static_slot, t('activerecord.models.static_slot.one') %><br />
```

Comment:
Use `StaticSlot.model_name.human` instead of directly querying the i18n key. It's a bit more readable, and maybe easier to maintain